packages/pocketbase/build.sh

```diff
@@ -0,0 +1,29 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/pocketbase/pocketbase
+TERMUX_PKG_DESCRIPTION="An open source Go backend"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION="0.8.0"
+TERMUX_PKG_SRCURL="https://github.com/pocketbase/pocketbase/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz"
+TERMUX_PKG_SHA256=3d5130f21d1d74a58b663b1b9946ce1fbf467a6ac5335d7acc51a396e93ea330
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+
+termux_step_make() {
+	termux_setup_golang
+
+	mkdir ./gopath
+	export GOPATH="$PWD/gopath"
+
+	cd $TERMUX_PKG_SRCDIR/examples/base
+
+	export GOBUILD=CGO_ENABLED=0
+	
+	go build \
+		-trimpath \
+		-o "pocketbase.bin" \
+		main.go
+}
+
+termux_step_make_install() {
+    install -m700 examples/base/pocketbase.bin "${TERMUX_PREFIX}"/bin/pocketbase
```

Comment:
Please use tabs for indentation, and add a newline at the the end

Reply:
Is that strictly required? I don't usually use vscode, but I'm using github codespaces to manage the repo. As you can see I updated the file but vscode stripped the new line at the end. If it's required I will push an update again.

Reply:
Tabs should be used as per [coding guidelines](https://github.com/termux/termux-packages/wiki/Coding-guideline). 

Pretty much all other files in the repo end with newline, I don't see why this one should be different

Reply:
Fixed